.github/workflows/publish-image-on-push-to-branch.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Publish image to GHCR on push to branch'
+
+on:
+  push:
+    branches-ignore:
+      - 'main'
+
+concurrency:
+  group: clammit-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build_and_push:
+    permissions:
+      packages: write
+      security-events: write
+      contents: read
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/image-publish-trivy.yaml@5.0.0
```

Comment:
could get updated to version 7, maintenance is easier if minor and bugfix version are not strictly defined